templates/patient_extension.js.erb

```diff
@@ -10,22 +10,6 @@
   extended_data: {},
 }, { id: false });
 
-const PatientSchema = DataElementSchema({
```

Comment:
Why was this object removed from here? It still seems to be in `Patient.js`

Reply:
This template is an extension on `mongoose_tmeplate.js.erb`, so it gets injected into what's already in that template--including most of this object. If you look in the full version of this extension, you'll see how it starts at halfway down this object, because it's an extension on it.

This was an accidental duplicate created sometime along the way. When updating the template one time I accidentally copy-pasted too much from `Patient.js`. This fixes that issue.